File: app/src/main/java/fr/free/nrw/commons/nearby/model/ResultTuple.java
package fr.free.nrw.commons.nearby.model;

public class ResultTuple {
    private final String type;
    private final String value;

    public ResultTuple(String type, String value) {
        this.type = type;
        this.value = value;
    }

    public ResultTuple() {
        this.type = "";
        this.value = "";
    }

    public String getType() {
        return type;
    }

    public String getValue() {
        return value;
    }
}

File: app/src/main/java/fr/free/nrw/commons/nearby/model/ResultTuple.kt
package fr.free.nrw.commons.nearby.model

import com.google.gson.annotations.SerializedName
import java.util.Locale

class ResultTuple {
    @SerializedName("xml:lang")
    val language: String
    val type: String
    var value: String

    constructor(lang: String, type: String, value: String) {
        this.language = lang
        this.type = type
        this.value = value
    }

    constructor() {
        language = Locale.getDefault().language
        type = ""
        value = ""
    }
}
